### DISCUSSION TITLE
Welcome to Lending-Bot Discussions! OK

### DISCUSSION THREAD

**Discussion by eros1006** [DedicatedSoftwareDev/Lending-Bot#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by websparkledev** [comment]
I think lending bot platform should be deployed on AWS or other clouding services in order to increase subscribers and users.
